sass/directives/_sections.scss

```diff
@@ -0,0 +1,20 @@
+@mixin logos-section {
+  background-color: $white;
+  padding-top: $section-padding;
+  padding-bottom: $section-padding;
+
+  .home__logos__headline {
+    font-size: 30px;
```

Comment:
I don't suppose there's a standard `header` `font-size` variable we could use or make here?